.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           PGHOST: 127.0.0.1
           PGUSER: ncce_test
           RAILS_ENV: test
-      - image: circleci/postgres:9.5-alpine
+      - image: cimg/postgres:9.5-alpine
```

Comment:
Did you find the list of images at https://circleci.com/developer/images/image/cimg/postgres ?
This might be an opportunity to upgrade the postgres to match the versions we use in dev and in heroku.